### ISSUE TITLE
Fails to create a workspace in window 10

### ISSUE THREAD

**Issue by gameracer** [fahamutech/bfast-tools#6]
### Step to reproduce
1. install bfast-tools `npm install -g bfast-tools`
2. run `bfast functions create demofunctions`
 You will get following errors

**Comment by joshuamshana** [comment]
me too it fail in cmd window 10
```
Microsoft Windows [Version 10.0.18362.476]
(c) 2019 Microsoft Corporation. All rights reserved.

C:\xxxxi>bfast
Usage: index [options] [command]

Options:
  -V, --version  output the version number
  -h, --help     output usage information

Commands:
  functions|fs   manage bfast cloud functions
  help [cmd]     display help for [cmd]

C:\Users\Udicti>bfast functions create demofaas
internal/modules/cjs/loader.js:800
    throw err;
    ^

Error: Cannot find module 'C:\Users\Udicti\index-functions'
[90m    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:797:15)[39m
[90m    at Function.Module._load (internal/modules/cjs/loader.js:690:27)[39m
[90m    at Function.Module.runMain (internal/modules/cjs/loader.js:1047:10)[39m
[90m    at internal/main/run_main_module.js:17:11[39m {
  code: [32m'MODULE_NOT_FOUND'[39m,
  requireStack: []
}

C:\xxxx>
```

**Comment by joshuamshana** [comment]
fixed in v0.2.1, upgrade by running `~$ npm install -g bfast-tools`
